openshift-ci/Dockerfile.tools

```diff
@@ -0,0 +1,9 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+ENV SHELLCHECK_VERSION=v0.7.0
+
+# Install shellcheck: https://github.com/koalaman/shellcheck
+RUN curl -L "https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK_VERSION}/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" -o shellcheck.tar.xz \
+    && tar -xvf ./shellcheck.tar.xz \
+    && mv ./shellcheck*/shellcheck /usr/local/bin \
+    && rm -rf ./shellcheck*
```

Comment:
What is the purpose of this (the reason I am asking is that I don't know :) )?

Reply:
This tool is used for giving warnings and suggestions when developing bash scripts (also available in vscode, which is pretty useful) and after talking to @mikenairn, for the start, we decided to use it in PR tests by OpenShift CI.
The container image is already available in our CI integr8ly namespace. Now I'm just waiting for https://github.com/openshift/release/pull/7637 to be merged.